client/app/api/course/LearningMap.js

```diff
@@ -0,0 +1,100 @@
+import BaseCourseAPI from './Base';
+
+export default class LearningMapAPI extends BaseCourseAPI {
+  /**
+   * Fetches all nodes in the learning map for the current user and course.
+   *
+   * @return {Promise}
+   * success response: {
+   *   nodes: Array.<{
+   *     id: string, unlocked: boolean, satisfiabilityType: string,
+   *     courseMaterialType: string, contentUrl: string,
+   *     children: Array.<{ id: string, is_satisfied: boolean }>,
```

Comment:
Will the nested property still be isSatisfied?

Reply:
Yup it's `isSatisfied` but I overlooked it when updating the comment. It's been updated in all method comments.